### PR TITLE
Daryl david/582/uk postcode formating 3

### PR DESCRIPTION
**Pull request by DarylMConway**

### CODE REVIEW

File: app/uk/gov/hmrc/gform/validation/ValidationService.scala [hunk 1]
```diff
@@ -503,7 +503,7 @@ class ComponentsValidator(
       case value :: Nil =>
         Map(fieldId -> errors(fieldValue, "must not be entered")).invalid
       case value :: rest =>
-        Map(fieldId -> errors(fieldValue, "must not be entered")).invalid // we don't support multiple values yet
+        Map(fieldId -> errors(fieldValue, "must not be entered")).invalid // we don't support multiple values yetcompile
```
Comment (josef-vlach) [inline app/uk/gov/hmrc/gform/validation/ValidationService.scala]: `yetcompile` seems like typo.

File: app/uk/gov/hmrc/gform/validation/ValidationService.scala [hunk 2]
```diff
@@ -522,6 +522,13 @@ class ComponentsValidator(
     }
   }
 
+  private def postcodeValidation(fieldValue: FormComponent, fieldId: FormComponentId)(xs: Seq[String]): ValidatedType =
+    (xs.filterNot(_.isEmpty), fieldId) match {
```
Comment (josef-vlach) [inline app/uk/gov/hmrc/gform/validation/ValidationService.scala]: No need to pattern match on `fieldId` as it is not used.

Comment (DarylMConway) [reply app/uk/gov/hmrc/gform/validation/ValidationService.scala]: i have removed unused feildId within the pattern matching

File: app/uk/gov/hmrc/gform/validation/ValidationService.scala (hunk 1 above)
Comment (DarylMConway) [reply app/uk/gov/hmrc/gform/validation/ValidationService.scala]: typo has been removed

**Pull request merged**